importers/db/dbschema.py

```diff
@@ -468,6 +468,19 @@ def _init_git_tables(self):
                                    "PRIMARY KEY fityli (file_modification_id, type, line_number) " \
                                    ") ENGINE=InnoDB DEFAULT CHARSET=utf8 ROW_FORMAT=DYNAMIC;"
 
+        # adding it here because "file_dependency" depends on "file" table creation.
+        # @todo: find a way to move the following table creation to separate section
+        #   make "extract_dependency_relations" API interface completely independent.
+        create_table_file_dependency = "CREATE TABLE file_dependency ( " \
```

Comment:
foreign keys are not needed, in gitana they are checked programmatically and there are no delete operations